src/components/Dashboard.tsx

```diff
@@ -112,9 +112,9 @@ const TemplateComponent: React.FC<{
       <div className="content">
         <div className="filters">
           <Label className="strong-label clickable">
-            <a href={`/applications?type=${code}&user-role=${userRole}`}>
+            <Link to={`/applications?type=${code}&user-role=${userRole}`}>
```

Comment:
This was a mistake. Bare `a` links don't work as they bypass React Router and force a full page reload.